schema/spacewalk/common/tables/rhnVirtualInstance.sql

```diff
@@ -28,6 +28,7 @@ CREATE TABLE rhnVirtualInstance
     uuid               VARCHAR2(128),
     confirmed          NUMBER(1)
                            DEFAULT (1) NOT NULL,
+    payg               CHAR(1) DEFAULT ('N') NOT NULL,
```

Comment:
As Joe has pointed out, AWS offers bare-metal PAYG instances as well, so this bit should not be in `rhnVirtualInstance`. I wonder if it makes sense to have a completely separate table (say `rhnServerPAYG`, with just a `server_id` column) or just add a column to `rhnServer`.

Reply:
Hmm, if there are bare-metal instances it might make sense to add this to rhnServer directly.

Reply:
IIUC even for those bare-metal instances, there would be also an entry on `rhnVirtualInstance` table for this particular server (as we use that entry to match with their possible virtual guests). This table is not actually storing only "virtual instances" but it also contains entries for the "virtualization hosts". So the current design will still work even with bare-metal instances. 

But, as you mentioned, it may make more sense to add this attribute directly in `rhnServer` table. 

In any case, I don't think it's worth to create a complete new table just for storing this new attribute.

Reply:
There would be other possible tables, but I am not sure if they are better or not.

`rhnServerInfo` - has the checkin counter and date.
`suseServer` - maybe not a good idea. It is an enhancement of rhnServer for easier schema merging with upstream. Long term I plan to remove it an merge the needed values with rhnServer.

So I see two options: either `rhnServer` or `rhnServerInfo`

Reply:
Ok, I've pushed the change to make the `payg` attribute part of the `rhnServer` table instead of `rhnVirtualInstance`.